content/applications/hr/payroll.rst

```diff
@@ -241,36 +396,36 @@ There are two default structure types configured in Odoo: *Employee* and *Worker
    :alt: List of all currently configured structure types available to use.
 
 New structure type
-------------------
+~~~~~~~~~~~~~~~~~~
 
 To make a new structure type, click the :guilabel:`New` button and a structure type form appears.
 Enter the information in the fields. Most fields are pre-populated, but all the fields can be
 modified.
 
-- :guilabel:`Structure Type`: Enter the name for the new structure type, such as 'Employee' or
-  'Worker'.
-- :guilabel:`Country`: Select the country that the new structure type applies to from the drop-down
+- :guilabel:`Structure Type`: enter the name for the new structure type, such as `Employee` or
+  `Worker`.
+- :guilabel:`Country`: select the country that the new structure type applies to from the drop-down
   menu.
-- :guilabel:`Wage Type`: Select what type of wage the new structure type will use, either
-  :guilabel:`Monthly Fixed Wage` or :guilabel:`Hourly Wage`. If the wage type is going to be used
-  for salaried employees who receive the same wage every pay period, select :guilabel:`Monthly Fixed
-  Wage`. If the wage type is going to be used for employees who receive wages based on how many
-  hours they worked during a pay period, select :guilabel:`Hourly Wage`.
-- :guilabel:`Default Scheduled Pay`: Select the typical pay schedule for the new structure type from
+- :guilabel:`Wage Type`: select what type of wage the new structure type will use, either
```

Comment:
I checked with both a 17,0, and a 17.1 (even though I used 17.0 for this doc) as well as my test DB, and they all say "Wage Type" not "Default Wage Type". Since I checked in 3 places, I assume when you saw that, something was done to the DB. This does seem to be the standard, from triple checking.